packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx

```diff
@@ -1,54 +1,57 @@
-import { type TenantInfo, TenantTag } from '@logto/schemas/models';
+import { type TenantInfo, type TenantTag } from '@logto/schemas/models';
 import classNames from 'classnames';
-import { useEffect, useState } from 'react';
+import { useContext, useEffect, useMemo, useState } from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 import { toast } from 'react-hot-toast';
 import { useTranslation } from 'react-i18next';
 
 import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import { useCloudSwr } from '@/cloud/hooks/use-cloud-swr';
 import AppError from '@/components/AppError';
 import AppLoading from '@/components/AppLoading';
 import PageMeta from '@/components/PageMeta';
 import SubmitFormChangesActionBar from '@/components/SubmitFormChangesActionBar';
 import UnsavedChangesAlertModal from '@/components/UnsavedChangesAlertModal';
-import useTenants from '@/hooks/use-tenants';
+import { TenantsContext } from '@/contexts/TenantsProvider';
 
 import DeleteCard from './DeleteCard';
 import DeleteModal from './DeleteModal';
 import ProfileForm from './ProfileForm';
 import * as styles from './index.module.scss';
 import { type TenantSettingsForm } from './types.js';
 
-const tenantProfileToForm = (tenant?: TenantInfo): TenantSettingsForm => {
+const tenantProfileToForm = (tenant: TenantInfo): TenantSettingsForm => {
+  const { name, tag } = tenant;
   return {
-    profile: { name: tenant?.name ?? 'My project', tag: tenant?.tag ?? TenantTag.Development },
+    profile: { name, tag },
```

Comment:
nit:
1. The function name would be better if it's a verb-object phrase, such as `convertTenantInfoToFormData`
2. Don't we need the fallback default values any more?

Reply:
If we can not get "current tenant", then we should not come to these pages.

Reply:
ok